src/components/App/App.js

```diff
@@ -13,9 +13,8 @@ class App extends Component {
           }
      }
 
-      filter = (event) => {
-      let searchTitle = event.target.value
-      this.setState({displayed: this.movies.filter(movie => movie.title.toLowerCase.includes(searchTitle.toLowerCase))
+      filter = (searchedTitle) => {
+      this.setState({displayed: this.state.movies.filter(movie => movie.title.toLowerCase().includes(searchedTitle.toLowerCase()))
```

Comment:
ooo I'd encourage you to run the filter _before_ the setState method, rather than within it. setState is actually an async function, so we should do our best to just give it the data it needs, rather than asking it to run a filter.